public/meta/preload.json

```diff
@@ -1,163 +1,15 @@
 {
   "items": [
     {
-      "url": "https://files.pythonhosted.org/packages/10/cb/f2ad4230dc2eb1a74edf38f1a38b9b52277f75bef262d8908e60d957e13c/blinker-1.9.0-py3-none-any.whl",
-      "contentType": "binary/octet-stream"
-    },
-    {
-      "url": "https://files.pythonhosted.org/packages/b6/cb/b86984bed139586d01532a587464b5805f12e397594f19f931c4c2fbfa61/tenacity-9.0.0-py3-none-any.whl",
-      "contentType": "binary/octet-stream"
-    },
-    {
-      "url": "https://cdn.jsdelivr.net/pyodide/v0.26.2/full/pyodide.js",
```

Comment:
It's suspicious that this one completely disappeared from the preload list, I'll investigate